src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const arr = people.filter(person =>
```

Comment:
use more meanfull name instead of arr

Reply:
if we use 'people' in getAvarageAge(people) where we use const arr? i`m confused((( at first it was clear, but more i try improve more i don`t understand(

Reply:
just don't name variables like `arr, obj` and so on. it will be much clearer to name `men`